### PR TITLE
fix docs (#5143)

### PR DESCRIPTION
**Pull request by LearningPawn**
1. 修复 docs/api/paddle/nn/BatchNorm1D_cn.rst、docs/api/paddle/nn/BatchNorm2D_cn.rst、docs/api/paddle/nn/BatchNorm3D_cn.rst 中文文档比英文文档少一个参数 use_global_stats 的问题； 为参数 data_format 的说明添加更为详尽的描述，同目录下 BatchNorm_cn.rst 一并添加相关描述；使中文文档与英文文档的 Shapes 表述一致；
2. 修复 docs/api/paddle/nn/functional/batch_norm_cn.rst 中文文档比英文文档少参数的问题，并为参数 data_format 的说明添加更为详尽的描述。

英文文档：https://github.com/PaddlePaddle/Paddle/pull/49044

### CODE REVIEW

File: docs/api/paddle/nn/functional/batch_norm_cn.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 batch_norm
 -------------------------------
 
-.. py:class:: paddle.nn.functional.batch_norm(x, running_mean, running_var, weight, bias, training=False, momentum=0.9, epsilon=1e-05, data_format='NCHW', name=None):
+.. py:class:: paddle.nn.functional.batch_norm(x, running_mean, running_var, weight, bias, training=False, momentum=0.9, epsilon=1e-05, data_format='NCHW', use_global_stats=None, name=None):
```
Comment (Ligoml) [inline docs/api/paddle/nn/functional/batch_norm_cn.rst]:
```suggestion
.. py:class:: paddle.nn.functional.batch_norm(x, running_mean, running_var, weight, bias, training=False, momentum=0.9, epsilon=1e-05, data_format='NCHW', use_global_stats=None, name=None)
```

File: docs/api/paddle/nn/functional/batch_norm_cn.rst [hunk 2]
```diff
@@ -17,9 +17,11 @@ batch_norm
     - **running_var** (Tensor) - 方差的 Tensor。
     - **weight** (Tensor) - 权重的 Tensor。
     - **bias** (Tensor) - 偏置的 Tensor。
+    - **training** (bool，可选) – 当该值为 True 时，表示为训练模式（train mode），即使用批数据计算并在训练期间跟踪全局均值和方差。为 False 时，表示使用预测模式（inference mode），即使用训练期间计算出的全局均值及方差计算。默认值为 False。
```
Comment (Ligoml) [inline docs/api/paddle/nn/functional/batch_norm_cn.rst]:
```suggestion
    - **training** (bool，可选) – 当该值为 True 时，表示为训练模式（train mode），即使用批数据计算并在训练期间跟踪全局均值和方差。为 False 时，表示使用推理模式（inference mode），即使用训练期间计算出的全局均值及方差计算。默认值为 False。
```

File: docs/api/paddle/nn/functional/batch_norm_cn.rst [hunk 3]
```diff
@@ -17,9 +17,11 @@ batch_norm
...
     - **momentum** (float，可选) - 此值用于计算 ``moving_mean`` 和 ``moving_var``。默认值：0.9。更新公式如上所示。
     - **epsilon** (float，可选) - 为了数值稳定加在分母上的值。默认值：1e-05。
-    - **data_format** (string，可选) - 指定输入数据格式，数据格式可以为“NC", "NCL", "NCHW" 或者"NCDHW"。默认值："NCHW"。
+    - **data_format** (string，可选) - 指定输入数据格式。数据格式可以为 ``"NC"``、``"NCL"``、``"NCHW"`` 或者 ``"NCDHW"``，其中 N 是批大小，C 是通道数，D 是特征深度，H 是特征高度，W 是特征宽度，L 是特征长度。默认值为 ``"NCHW"``。
```
Comment (Ligoml) [inline docs/api/paddle/nn/functional/batch_norm_cn.rst]:
```suggestion
    - **data_format** (str，可选) - 指定输入数据格式。数据格式可以为 ``"NC"``、``"NCL"``、``"NCHW"`` 或者 ``"NCDHW"``，其中 N 是批大小，C 是通道数，D 是特征深度，H 是特征高度，W 是特征宽度，L 是特征长度。默认值为 ``"NCHW"``。
```

File: docs/api/paddle/nn/BatchNorm2D_cn.rst [hunk 4]
```diff
@@ -3,7 +3,7 @@
 BatchNorm2D
 -------------------------------
 
-.. py:class:: paddle.nn.BatchNorm2D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCHW', name=None):
+.. py:class:: paddle.nn.BatchNorm2D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCHW', use_global_stats=None, name=None):
```
Comment (Ligoml) [inline docs/api/paddle/nn/BatchNorm2D_cn.rst]:
```suggestion
.. py:class:: paddle.nn.BatchNorm2D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCHW', use_global_stats=None, name=None)
```

File: docs/api/paddle/nn/BatchNorm3D_cn.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@
 BatchNorm3D
 -------------------------------
 
-.. py:class:: paddle.nn.BatchNorm3D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCDHW', name=None):
+.. py:class:: paddle.nn.BatchNorm3D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCDHW', use_global_stats=None, name=None):
```
Comment (Ligoml) [inline docs/api/paddle/nn/BatchNorm3D_cn.rst]:
```suggestion
.. py:class:: paddle.nn.BatchNorm3D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCDHW', use_global_stats=None, name=None)
```

**Review by Ligoml — COMMENTED** [review]
good job！还有几处小问题需要修改

**Comment by LearningPawn** [comment]
@Ligoml 已经根据您的意见进行了修改，感谢您的 Review 与指导！

**Review by Ligoml — APPROVED** [review]
good job！LGTM for docs
